corehq/apps/hqcase/dbaccessors.py

```diff
@@ -57,3 +57,23 @@ def get_case_types_for_domain(domain):
         if case_type:
             case_types.append(case_type)
     return case_types
+
+
+def get_case_ids_in_domain_by_owner(domain, owner_id=None, owner_id__in=None,
```

Comment:
Maybe a comment here about the usage of `owner_id` `owner_id__in` and `closed` here?